bedrock/firefox/templates/firefox/releases/release-notes.html

```diff
@@ -1,166 +1,438 @@
 {# This Source Code Form is subject to the terms of the Mozilla Public
- # License, v. 2.0. If a copy of the MPL was not distributed with this
- # file, You can obtain one at http://mozilla.org/MPL/2.0/. #}
-
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/. #}
 {% extends "firefox/base-resp.html" %}
-
 {% block page_title_prefix %}{% endblock %}
+{% from "macros.html" import fxfamilynav, send_to_device with context %}
```

Comment:
Doesn't look like these macros are used anywhere on the page. Any reason you have them imported here? Can we get rid of this line?